03_launch_mgmt_cluster.sh

```diff
@@ -105,10 +107,13 @@ function clone_repos() {
 
 function patch_clusterctl(){
   pushd "${CAPM3PATH}"
-  if [ -n "${CAPM3_LOCAL_IMAGE}" ]; then
+  if [ -n "${CAPM3_LOCAL_IMAGE}" ] && [ -n "${BAREMETAL_OPERATOR_LOCAL_IMAGE}" ]; then
     CAPM3_IMAGE_NAME="${CAPM3_LOCAL_IMAGE##*/}"
+    BMO_IMAGE_NAME="${BAREMETAL_OPERATOR_LOCAL_IMAGE##*/}"
     export MANIFEST_IMG="192.168.111.1:5000/localimages/$CAPM3_IMAGE_NAME"
+    export MANIFEST_IMG_BMO="192.168.111.1:5000/localimages/$BMO_IMAGE_NAME"
     export MANIFEST_TAG="latest"
+    export MANIFEST_TAG_BMO="latest"
```

Comment:
those two images should not be mixed. you need to duplicate the condition, one for CAPM3, one for BMO

Reply:
Updated